src/TinyMCEInput.js

```diff
@@ -82,7 +82,8 @@ var TinyMCEInput = React.createClass({
       pollInterval: 1000,
       textareaProps: {},
       otherEventHandlers: {},
-      onChange: function() {}
+      onChange: function() {},
+      component: 'textinput',
```

Comment:
I tested this in my browser, and it seemed to work. I did not notice the change to textinput. I will take a look this evening and see what's going on.